### PR TITLE
BDOG-726: Update port, and move test endpoint route

### PR DESCRIPTION
**Pull request by cjwebb**

### CODE REVIEW

File: conf/testOnlyDoNotUseInAppConf.routes [hunk 1]
```diff
@@ -11,3 +11,4 @@
 
 # Add all the application routes to the prod.routes file
 ->         /                          prod.Routes
+GET        /test-only/                uk.gov.hmrc.trackingconsentfrontend.controllers.TestController.test
```
Comment (matthewmascord) [inline conf/testOnlyDoNotUseInAppConf.routes]: Will this still work in QA?

File: conf/application.conf [hunk 2]
```diff
@@ -109,5 +109,5 @@ assets {
 }
 
 cookie-banner {
-    port = 9000
+    port = 12345
```
Comment (matthewmascord) [inline conf/application.conf]: This has actually just gone in already.

File: conf/testOnlyDoNotUseInAppConf.routes (hunk 1 above)
Comment (cjwebb) [reply conf/testOnlyDoNotUseInAppConf.routes]: Its configurable with the `application.router` config property.
Do we need it to work in QA?

Comment (matthewmascord) [reply conf/testOnlyDoNotUseInAppConf.routes]: Well yes ideally, for demonstration purposes?

**Pull request merged**